README.md

```diff
@@ -14,6 +14,14 @@ To configure this application to work with Trello's API, you need to update the
 
 Also ensure your board specified above contains three lists: "To Do", "Doing" and "Done". 
 
+### Unit Test Configuration
```

Comment:
This is good, but we should add some clear simple steps to actually run the tests ideally without an IDE (i.e. `poetry run pytest`). We should also mention any dependencies, e.g. chrome.